crates/cairo-lint-core/src/lints/double_comparison.rs

```diff
@@ -6,6 +6,8 @@ use cairo_lang_syntax::node::ids::SyntaxStablePtrId;
 use cairo_lang_syntax::node::{TypedStablePtr, TypedSyntaxNode};
 
 pub const SIMPLIFIABLE_COMPARISON: &str = "This double comparison can be simplified.";
+pub const IMPOSSIBLE_COMPARISON: &str =
+    "The condition contains contradictory comparisons, meaning no value can satisfy both conditions at once.";
```

Comment:
this is redundant with `pub const CONTRADICTORY_COMPARISON: &str = "This double comparison is contradictory and always false.";`